docs/whats_new/version_0_24_0.rst

```diff
@@ -0,0 +1,8 @@
+Version 0.24.0
+--------------
+
+Search size for rtt segger
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Added the possibility to search in a given range for the block of the device
+starting from the specified block address.
```

Comment:
```suggestion
Version 0.24.0
--------------

Search size for rtt segger
^^^^^^^^^^^^^^^^^^^^^^^^^^

Added the possibility to search in a given range for the block of the device
starting from the specified block address.

Tags in global-config
^^^^^^^^^^^^^^^^^^^^^^^^^^
....?

Overall improvement
^^^^^^^^^^^^^^^^^^^^^^^^^^

For this release, we focused on improving internal functionalities:

* ...

```

Reply:
Pleas take a look at the commits to see if I missed something

Reply:
Added improvements and feat